user_data/strategies/test_strategy.py

```diff
@@ -253,6 +253,15 @@ def populate_indicators(self, dataframe: DataFrame, metadata: dict) -> DataFrame
         dataframe['ha_low'] = heikinashi['low']
         """
 
+	# Retrieve best bid and best ask
+        # ------------------------------------
+        """
+        ob = self.dp.orderbook(metadata['pair'], 1)
+        print(ob)
```

Comment:
Super, thanks!

This print() should be commented out for merging to the freqtrade codebase.
Or removed.